docs/docusaurus/docs/cloud/deploy_gx_agent.md

```diff
@@ -1,6 +1,6 @@
 ---
-sidebar_label: 'Deploy the GX Agent'
-title: 'Deploy the GX Agent'
+sidebar_label: '(Optional) Deploy the GX Agent'
+title: '(Optional) Deploy the GX Agent'
```

Comment:
I agree with @Rachel-Reverie, the page title should not begin with `(Optional)`.